pkg/api/server/register_containers.go

```diff
@@ -301,7 +316,8 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//  - in: query
 	//    name: detachKeys
 	//    type: string
-	//    description: needs description
+	//    description: "Override the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [, , or _."
```

Comment:
The value of description should not be quoted.

Reply:
it has to be because something in there murders the go swagger generation

Reply:
Learn something new every day.